docs/reference/snapshot-restore/take-snapshot.asciidoc

```diff
@@ -272,10 +272,13 @@ policy. Other snapshots don't count toward the policy's retention limits.
 [[snapshot-retention-limits]]
 ==== Snapshot retention limits
 
-While not a hard limit, a snapshot repository shouldn't contain more than
-{max-snapshot-count} snapshots at a time. This ensures the repository's metadata
-doesn't grow to a size that may destabilize the master node. We recommend you
-set up your {slm-init} policy's retention rules to enforce this limit.
+We recommend you include retention rules in your {slm-init} policy to delete
+snapshots you no longer need.
+
+A snapshot repository can safely scale to thousands of snapshots. However, to
+manage its metadata, a large repository requires more memory and disk space on
```

Comment:
"disk space" is not true, sorry that was my unfortunate wording above. It's really just memory that's the limiting factor here.

Reply:
Thanks for the correction.